src/Aspire.Hosting.Docker/Aspire.Hosting.Docker.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageTags>aspire hosting kubernetes</PackageTags>
     <Description>Kubernetes publishing for .NET Aspire.</Description>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
```

Comment:
required due to k8s client not being signed

Reply:
Why is the k8s client a reference here?

Reply:
It was 50/50 on the best way to take the writer.

YamlDotNet doesn't expose a stream based writer like STJ does. It has the concept of an Emitter, but its terrible to use, and results in a massive amount of code to do very little.
Rather than working with just a Dictionary<string, object> at the root, I was just using it as it was used in aspirate, where I'm reusing k8s types, and outputting those transformer as compose entries. However plan is to introduce strong types for sections of the compose and directly serialize those once a Compose object is created